src/style.css

```
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2) {
  background-image: url(images/star-active.svg);
  background-repeat: no-repeat;
}
```
combine all these selectors with a comma